README.md

```diff
@@ -1,9 +1,45 @@
+|3|[Building the model layers](pytorch/building_the_model_layer.ipynb)|[![Open in SageMaker Studio Lab](https://studiolab.sagemaker.aws/studiolab.svg)](https://studiolab.sagemaker.aws/import/github/Aisuko/generative-ai/blob/main/pytorch/building_the_model_layer.ipynb)|[![Kaggle](https://kaggle.com/static/images/open-in-kaggle.svg)](https://www.kaggle.com/aisuko/building-the-model-layers)|
+|4|[Automatic differentiation](pytorch/automatic_differentiation.ipynb)|[![Open in SageMaker Studio Lab](https://studiolab.sagemaker.aws/studiolab.svg)](https://studiolab.sagemaker.aws/import/github/Aisuko/generative-ai/blob/main/pytorch/automatic_differentiation.ipynb)|[![Kaggle](https://kaggle.com/static/images/open-in-kaggle.svg)](https://www.kaggle.com/aisuko/automatic-differentiation)|
+|5|[About the optimization loop](pytorch/about_the_optimization_loop.ipynb)|[![Open in SageMaker Studio Lab](https://studiolab.sagemaker.aws/studiolab.svg)](https://studiolab.sagemaker.aws/import/github/Aisuko/generative-ai/blob/main/pytorch/about_the_optimization_loop.ipynb)|[![Kaggle](https://kaggle.com/static/images/open-in-kaggle.svg)](https://www.kaggle.com/aisuko/about-the-optimization-loop)|
+|6|[Load and run model predictions](pytorch/load_and_run_model_predictions.ipynb)|[![Open in SageMaker Studio Lab](https://studiolab.sagemaker.aws/studiolab.svg)](https://studiolab.sagemaker.aws/import/github/Aisuko/generative-ai/blob/main/pytorch/load_and_run_model_predictions.ipynb)|
+|7|[The full model building process](pytorch/the_full_model_building_process.ipynb)|[![Open in SageMaker Studio Lab](https://studiolab.sagemaker.aws/studiolab.svg)](https://studiolab.sagemaker.aws/import/github/Aisuko/generative-ai/blob/main/pytorch/the_full_model_building_process.ipynb)|[![Kaggle](https://kaggle.com/static/images/open-in-kaggle.svg)](https://www.kaggle.com/aisuko/the-full-model-building-process)|
+
+## Computer vision with PyTorch
+|No|Title|Open in Studio Lab|Open in Kaggle|
+|---|---|---|---|
+|1|[Introduction to CV with PyTorch](pytorch/computer_vision/introduction_to_cv_with_pytorch.ipynb)|[![Open in SageMaker Studio Lab](https://studiolab.sagemaker.aws/studiolab.svg)](https://studiolab.sagemaker.aws/import/github/Aisuko/generative-ai/blob/main/pytorch/computer_vision/introduction_to_cv_with_pytorch.ipynb)|[![Kaggle](https://kaggle.com/static/images/open-in-kaggle.svg)](https://www.kaggle.com/code/aisuko/introduction-to-computer-vision-with-pytorch)|
+|2|[Training a simple sense neural network](pytorch/computer_vision/training_a_simple_cnn.ipynb)|[![Open in SageMaker Studio Lab](https://studiolab.sagemaker.aws/studiolab.svg)](https://studiolab.sagemaker.aws/import/github/Aisuko/generative-ai/blob/main/pytorch/computer_vision/training_a_simple_cnn.ipynb)|[![Kaggle](https://kaggle.com/static/images/open-in-kaggle.svg)](https://www.kaggle.com/code/aisuko/training-a-simple-dense-neural-network)|
```

Comment:
The Kaggle link we should replace them to 

(https://kaggle.com/kernels/welcome?src=https://github.com/aisuko/pytorch/the_full_model_building_process.ipynb). This will allow anyone who open a new notebook on Kaggle rather than open mine directly

Reply:
i dont think this link works, please double-check it

Reply:
Sorry, https://kaggle.com/kernels/welcome?src=https://github.com/Aisuko/generative-ai/blob/main/pytorch/what_are_tensors.ipynb